assessments/0002-notary-project.md

```diff
@@ -0,0 +1,176 @@
+
+## Introduction
+
+This document assesses the quality and completeness of a project's documentation and website (if present).
+
+This document:
+
+- Measures existing documentation quality against the CNCF’s standards
+- Recommends specific and general improvements
+- Provides examples of great documentation as reference 
+- Identifies key improvements with the largest return on investment
+
+## How this document works
+
+The assessment is divided into three sections: 
+
+- **Project documentation:** for end users of the project; aimed at people who intend to use it
+- **Contributor documentation:** for new and existing contributors to the project
+- **Website:** branding, website structure, and maintainability
+
+Each section rates content based on different [criteria](criteria.md).
+
+## Project documentation 
+
+ Criteria: 
+ - 1 = (Is not present or requires significant work) 
+ - 3 = (is present, but needs work) 
+ - 5 = (is executed extremely well or no improvement required) |
+
+### V1
+
+| Criteria                   | 1   | 2   | 3   | 4   | 5   |
+| ---                        | --- | --- | --- | --- | --- |
+| Information architecture   |     |     | ✅  |     |     |
+| New user content           |     |     | ✅  |     |     |
+| Content maintainability    |     | ✅  |     |     |     |
+| Content creation processes |     |     | ✅  |     |     |
+
+### V2
+
+| Criteria                   | 1   | 2   | 3   | 4   | 5   |
+| ---                        | --- | --- | --- | --- | --- |
+| Information architecture   |     |     | ✅  |     |     |
+| New user content           | ✅  |     |     |     |     |
+| Content maintainability    | ✅  |     |     |     |     |
+| Content creation processes | ✅  |     |     |     |     |
+
+### Comments
+
+There are two versions of notary that need to be supported with documentation.
+- V1 is housed with [The Update Framework](https://github.com/theupdateframework/notary), and the docs are included as a set of markdown files in the [/docs folder](https://github.com/theupdateframework/notary/tree/master/docs)
+- V2 is housed with The Notary Project (https://github.com/notaryproject/notaryproject) 
+
+**Information Architecture**: 
+- As there is no website as yet, and development is still ongoing, V2's Project Documentation is all contained within markdown files in the notaryproject repo.
+
+**New user content**:
+- What is there does a good job of explaining the project, but doesn't provide many docs focused on new users.
+
+**Content maintainability**:
+- The docs currently aren't versioned or localized, and is depending on GitHub for search.
+
+**Content creation processes**:
+- There are links to community discussions, but as there is no website yet there is no space to point to regarding docs creation.
+
+### Recommendations
+
+V1 Docs will likely need to be reworked once an information architecture is setup for the new site. The new site IA can likely be informed by the existing V1 docs.
```

Comment:
In the spirit of "anyone should be able to pick up this work, not just you": spell it out.

"As there is no existing information architecture, we recommend the following, based on the v1 docs: 

- Overview
- Getting Started
- Some lovely concept
   - Some lovely concept or task
   - Another lovely task 
- Another lovely concept  "

.. etc.